content/applications/sales/sales/amazon_connector/setup.rst

```diff
@@ -1,104 +1,144 @@
+   synchronized, and automatically made available.
+
+After saving, the field in the :guilabel:`Credentials` tab is replaced by a :guilabel:`Link with
+Amazon` button.
+
+.. image:: setup/amazon-accounts-form-link-button.png
+   :align: center
+   :alt: A typical Amazon Account form page and Link with Amazon button in Odoo Sales.
+
+Clicking that button redirects to either the Amazon login page, or directly to the required consent
+page, if the user is already logged in to Amazon.
+
+On this page, confirm that Amazon is allowed to give Odoo access to the account and related data.
+
+Upon confirmation, Amazon returns the user to Odoo, and the account has been registered.
+
+With the account successfully registered, the marketplaces available to this specific account are
+synchronized and listed under the :guilabel:`Marketplaces` tab.
+
+If desired, remove items from the list of synchronized marketplaces to disable synchronization.
 
 Match database products in Amazon
 =================================
 
 When an Amazon order is synchronized, up to three sales order items are created in Odoo for each
-product sold on Amazon: one for the marketplace product, one for the shipping charges (if any) and
+product sold on Amazon: one for the marketplace product, one for the shipping charges (if any), and
 one for the gift wrapping charges (if any).
 
 .. _amazon/matching:
 
 The selection of a database product for a sales order item is done by matching its
-**internal reference** with the **SKU** for marketplace items, the **shipping code** for delivery
-charges, and the **gift wrapping** code for gift wrapping charges.
+:guilabel:`Internal Reference` with the *SKU* for marketplace items, the *Shipping Code* for
+delivery charges, and the *Gift Wrapping* code for gift wrapping charges.
+
+For marketplace products, pairings are saved as *Amazon Offers*, which are listed under the
+:guilabel:`Offers` button on the account form.
+
+.. image:: setup/amazon-offers-button.png
+   :align: center
+   :alt: The Amazon Offers smart button on the account form in Odoo Sales.
 
-For marketplace products, pairings are saved as **Amazon Offers** which are listed under the
-**Offers** stat button on the account form. Offers are automatically created when the pairing is
-established and are used for subsequent orders to lookup SKUs. If no offer with a matching SKU is
-found, :ref:`the internal reference is used instead <amazon/matching>`.
+Offers are automatically created when the pairing is established, and they're used for subsequent
+orders to lookup SKUs. If no offer with a matching SKU is found, :ref:`the internal reference is
+used instead <amazon/matching>`.
 
 .. tip::
-   It is possible to force the pairing of a marketplace item with a specific product by changing
+   It's possible to force the pairing of a marketplace item with a specific product, by changing
    either the product or the SKU of an offer. The offer can be manually created if it was not
```

Comment:
What are the steps to do this?

Reply:
Again, if I had access to an actual Amazon Seller account on Marketplace, I'd likely be able to break down the steps required to do this. However, given that I am only able to gather what I can from previously-written documentation context, I *believe* this is, essentially, suggesting that the Product Name or the SKU can be directly edited to ensure they match in this instance. I added a brief mention of that to the end of the sentence, which (I hope) clears things up a bit. But, if you think this still needs a bit of work, just let me know, and I'll happily change it to whatever you deem best. Thanks!

Reply:
From amazon account you can find the list of offers with this smart button
![image](https://github.com/odoo/documentation/assets/100136841/339ed233-c61c-4ad3-805b-60df38fbd88e)

Then you can create, or edit offer to map odoo product with amazon SKU
![image](https://github.com/odoo/documentation/assets/100136841/a31aa211-6f19-46e6-859d-9adc1b66f301)